python/lsst/afw/geom/detail/frameSetUtils.cc

```diff
@@ -55,10 +55,11 @@ PYBIND11_PLUGIN(frameSetUtils) {
         return nullptr;
     }
 
-    mod.def("getSkyFrame", getSkyFrame, "frameSet"_a, "index"_a, "copy"_a);
-    mod.def("makeTanWcsMetadata", makeTanWcsMetadata, "crpix"_a, "crval"_a, "cdMatrix"_a);
+    mod.def("makeSimpleWcsMetadata", makeSimpleWcsMetadata, "crpix"_a, "crval"_a, "cdMatrix"_a,
+            "projection"_a = "TAN");
     mod.def("readFitsWcs", readFitsWcs, "metadata"_a, "strip"_a = true);
```

Comment:
Why are members of `::detail` being exported?

Reply:
For unit tests